dataprofiler/labelers/char_load_tf_model.py

```diff
@@ -351,9 +363,9 @@ def fit(
             if reset_weights:
                 self.reset_weights()
 
-        history = defaultdict()
-        f1 = None
-        f1_report = []
+        history: Dict = defaultdict()
+        f1: Optional[float] = None
+        f1_report: Dict = {}
```

Comment:
```suggestion
        f1_report: Optional[Dict] = None
```

Maybe this so that if it does end up returning `{}` we can just check `if not f1_report and not f1` instead of having to check for a blank dict()?

Reply:
also I think recommendation is to use non mutable default values... `dict` is a mutable type. So I think most folks just set defaults to `None`

Reply:
good idea

Reply:
That's the case if it is in the function definition. Not when being defined within the func itself.